documents/operating.md

```diff
@@ -21,10 +21,100 @@ SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
 SECTION 2: 🤝🤝🤝 Partnership 🤝🤝🤝
 ---------------------------------------------------------------------
-- [ ] Types of involvement
-- [ ] Roles and responsibilities
-- [ ] Adding partners
-- [ ] Removing partners
+- [x] Types of involvement
+- [x] Roles and responsibilities
+- [x] Adding partners
+- [x] Removing partners
+
+### Types of Involvement
+1. **Corporate Overlord:** Corporate Overlords of BIFFUD own part of the company.
+They have votes during plotting sessions and can review pull requests to `/corporate`.
+
+2. **Members:** members can have ownership over BIFFUD projects. All Corporate Overlords
+are also members.  Members are entitled to attend BIFFUD meetings.
+
+3. **Minions:** minions are contributing participants in BIFFUD projects. They
+might contribute via code, cash, or expertise.  They are entitled to attend BIFFUD
+meetings.
+
```

Comment:
I suggest we separate membership / minion logistics from specific projects; I think someone should be able to join BIFFUD as a member or minion without being associated with (or otherwise beholden to) a specific project.